src/app/components/fee-row/components/fee-estimate-item.tsx

```diff
@@ -33,7 +35,15 @@ export function FeeEstimateItem(props: FeeEstimateItemProps) {
     >
       <SpaceBetween flexGrow={1}>
         <Caption ml="2px">{labels[index]}</Caption>
-        {visible ? <></> : <FiChevronDown />}
+        {visible ? (
+          isSelected ? (
+            <FiCheck color={color('accent')} size="14px" />
+          ) : (
+            <></>
+          )
+        ) : (
+          <FiChevronDown />
+        )}
```

Comment:
Nested ternaries to be used sparingly, can be hard to read

Reply:
I will simplify this before I merge it.